metadata-ingestion/src/datahub/sdk/lineage_client.py

```diff
@@ -1,27 +1,35 @@
 from __future__ import annotations
 
 import difflib
-import logging
 from typing import TYPE_CHECKING, List, Literal, Optional, Set, Union
 
-import datahub.metadata.schema_classes as models
```

Comment:
removed this and imported classes directly for consistency

Reply:
I actually like using `models.WhateverClass`

Reply:
updated